src/css/components/icon.scss

```diff
@@ -61,8 +82,32 @@
     fill: $color-error;
   }
 
-  &.icon-alt {
-    fill: $color-primary-alt-light;
+  &.icon-active {
+    fill: $color-blue;
+  }
+}
+
+.icon-chevron {
```

Comment:
I'd like to punt this until we have more UI SVGs — I don't know yet how they would best be classed